MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -1679,7 +1697,7 @@ private void updateLayerOffsets(boolean forceUpdate) {
       lastCameraPosition = position;
       locationLayerController.cameraBearingUpdated(position.bearing);
       locationLayerController.cameraTiltUpdated(position.tilt);
-      updateAccuracyRadius(getLastKnownLocation(), true);
+      updateAccuracyRadius(getLastKnownLocation(), true, true);
```

Comment:
Replace the last param with `forceUpdate`

Reply:
i'll fix that in a min

Reply:
@Chaoba it's done